lib/firefox_profile.js

```diff
@@ -630,7 +630,21 @@ FirefoxProfile.prototype._addonDetails = function(addonPath, cb) {
   // If not found, this is probably a jetpack style addon, so parse
   // the `package.json` file for addon details
   catch (e) {
-    var manifest = require(path.join(addonPath, 'package.json'));
+    try {
+      var manifest = require(path.join(addonPath, 'package.json'));
+    } catch (e) {
+      // still not found, maybe this is a webextension, so parse the
+      // `manifest.json` file for addon details
+      var webExtManifest = JSON.parse(fs.readFileSync(
+        path.join(addonPath, 'manifest.json')));
```

Comment:
should we fail if `manifest.json` is not present? Is `manifest.json` a required file in a webextension?